component/site/views/flat/assets/css/events_css.css

```diff
@@ -999,4 +999,7 @@ div#jumpto form {text-align:center; width: 93%;margin-top:10px;}
     width: 195px !important;
     text-align: left;
     display: block;
-}
\ No newline at end of file
+}
+
+/** Make sure choosen list is under input **/
+#jevents .chzn-container.chzn-container-single .chzn-results {clear:both;}
```

Comment:
In which context is this?  Are there no situations where the <select> replacement needs to be inline with the label?